[debug] [2021-04-14T03:39:29.319Z] ----------------------------------------------------------------------
[debug] [2021-04-14T03:39:29.322Z] Command:       /usr/local/Cellar/node/14.9.0/bin/node /usr/local/bin/firebase deploy --only hosting
[debug] [2021-04-14T03:39:29.322Z] CLI Version:   8.12.0
[debug] [2021-04-14T03:39:29.323Z] Platform:      darwin
[debug] [2021-04-14T03:39:29.323Z] Node Version:  v14.9.0
[debug] [2021-04-14T03:39:29.325Z] Time:          Tue Apr 13 2021 22:39:29 GMT-0500 (Central Daylight Time)
[debug] [2021-04-14T03:39:29.325Z] ----------------------------------------------------------------------
[debug] [2021-04-14T03:39:29.325Z] 
[debug] [2021-04-14T03:39:29.345Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2021-04-14T03:39:29.345Z] > authorizing via signed-in user
[debug] [2021-04-14T03:39:29.346Z] [iam] checking project valion-1ab86 for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2021-04-14T03:39:29.348Z] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[debug] [2021-04-14T03:39:29.349Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[debug] [2021-04-14T03:39:29.523Z] <<< HTTP RESPONSE 200 {"cache-control":"no-cache, no-store, max-age=0, must-revalidate","pragma":"no-cache","date":"Wed, 14 Apr 2021 03:39:29 GMT","expires":"Mon, 01 Jan 1990 00:00:00 GMT","content-type":"application/json; charset=utf-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","server":"scaffolding on HTTPServer2","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-29=\":443\"; ma=2592000,h3-T051=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2021-04-14T03:39:29.534Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/valion-1ab86:testIamPermissions  
 {"permissions":["firebase.projects.get","firebasehosting.sites.update"]}
[debug] [2021-04-14T03:39:29.945Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=UTF-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Wed, 14 Apr 2021 03:39:29 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","server-timing":"gfet4t7; dur=260","alt-svc":"h3-29=\":443\"; ma=2592000,h3-T051=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2021-04-14T03:39:29.948Z] [iam] error while checking permissions, command may fail: FirebaseError: Authorization failed. This account is missing the following required permissions on project valion-1ab86:

  firebase.projects.get
  firebasehosting.sites.update
[debug] [2021-04-14T03:39:29.949Z] >>> HTTP REQUEST GET https://firebase.googleapis.com/v1beta1/projects/valion-1ab86  
 
[debug] [2021-04-14T03:39:31.345Z] <<< HTTP RESPONSE 403 {"vary":"X-Origin, Referer, Origin,Accept-Encoding","content-type":"application/json; charset=UTF-8","date":"Wed, 14 Apr 2021 03:39:31 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-29=\":443\"; ma=2592000,h3-T051=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2021-04-14T03:39:31.345Z] <<< HTTP RESPONSE BODY {"error":{"code":403,"message":"The caller does not have permission","status":"PERMISSION_DENIED"}}
[debug] [2021-04-14T03:39:31.347Z] HTTP Error: 403, The caller does not have permission
[debug] [2021-04-14T03:39:31.349Z] FirebaseError: HTTP Error: 403, The caller does not have permission
    at module.exports (/usr/local/lib/node_modules/firebase-tools/lib/responseToError.js:38:12)
    at Request._callback (/usr/local/lib/node_modules/firebase-tools/lib/api.js:41:35)
    at Request.self.callback (/usr/local/lib/node_modules/firebase-tools/node_modules/request/request.js:185:22)
    at Request.emit (events.js:314:20)
    at Request.EventEmitter.emit (domain.js:486:12)
    at Request.<anonymous> (/usr/local/lib/node_modules/firebase-tools/node_modules/request/request.js:1154:10)
    at Request.emit (events.js:314:20)
    at Request.EventEmitter.emit (domain.js:486:12)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/firebase-tools/node_modules/request/request.js:1076:12)
    at Object.onceWrapper (events.js:420:28)
    at IncomingMessage.emit (events.js:326:22)
    at IncomingMessage.EventEmitter.emit (domain.js:486:12)
    at endReadableNT (_stream_readable.js:1244:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
[error] 
[error] Error: Failed to get Firebase project valion-1ab86. Please make sure the project exists and your account has permission to access it.
